tests/capture/test_nested_plxpr.py

```diff
@@ -159,3 +161,111 @@ def qfunc(wire):  # x is closure variable and a tracer
 
         assert len(q) == 1
         qml.assert_equal(q.queue[0], qml.adjoint(qml.RX(2.5, 2)))
+
+
+class TestCtrlQfunc:
+    """Tests for the ctrl primitive."""
+
+    def test_operator_type_input(self):
+        """Test that an operator type can be the callable."""
+
+        def f(x, w):
+            return qml.ctrl(qml.RX, 1)(x, w)
+
+        plxpr = jax.make_jaxpr(f)(0.5, 0)
+
+        with qml.queuing.AnnotatedQueue() as q:
+            out = jax.core.eval_jaxpr(plxpr.jaxpr, plxpr.consts, 1.2, 2)
+
+        assert f(0.5, 0) is None
+        assert out == []
+        expected = qml.ctrl(qml.RX(1.2, 2), 1)
+        qml.assert_equal(q.queue[0], expected)
+
+        assert plxpr.eqns[0].primitive == ctrl_prim
+        assert plxpr.eqns[0].params["control_values"] == [True]
+        assert plxpr.eqns[0].params["n_control"] == 1
+        assert plxpr.eqns[0].params["work_wires"] is None
+
+    def test_dynamic_control_wires(self):
+        """Test that control wires can be dynamic."""
+
+        def f(w1, w2, w3):
+            return qml.ctrl(qml.X, (w2, w3))(w1)
+
+        plxpr = jax.make_jaxpr(f)(4, 5, 6)
+
+        with qml.queuing.AnnotatedQueue() as q:
+            out = jax.core.eval_jaxpr(plxpr.jaxpr, plxpr.consts, 1, 2, 3)
+
+        assert out == []
```

Comment:
```suggestion
        assert out is None
```

Reply:
Again, can't override the behavior of `jax.core.eval_jaxpr`.